### ISSUE TITLE
Text Classification for Serbian Science Journals

### ISSUE THREAD

**Issue by milossdjuric** [ftn-ai-lab/ori-2023-ra#11]
**Student**
- Miloš Đurić RA217/2020

**Asistent**
- Aleksandar Vujinović

**Problem koji se rešava**
- Cilj projekta je korišćenje algoritama za klasifikaciju delova teksta iz srpskih naučnih časopisa po različitim kategorijama
- Kategorije mogu biti sledeće:
&nbsp; - Arhitektura i građevinarstvo
&nbsp; - Biologija
&nbsp; - Ekonomija
&nbsp; - Geonauka
&nbsp; - Hemija
&nbsp; - Informatika i elektrotehnika
&nbsp; - Matematika
&nbsp; - Medicina
&nbsp; - Poljoprivreda

**Algoritmi** 
- Multinomial Naive Bayes, SVM, KNN

**Podaci koji se koriste**
- Koriste se podaci sa sajta: https://scindeks.ceon.rs/

**Metrika za merenje performansi**
- Performanse će se meriti sa accuracy i macro average metrikama

**Validacija rešenja**
- Dataset će biti raspodeljen 70% na training set, 15% na validation set i 15% na test set

**Comment by alek98** [comment]
Tema je odobrena. Srećan rad.

**Comment by milossdjuric** [comment]
Link do repozitorijuma: https://github.com/milossdjuric/Text-Classification-for-Serbian-Science-Journals
